### PR TITLE
gui/controllers dialog: Align text in controllers dialog

### PR DESCRIPTION
**Pull request by Croden1999**
# About
- gui/controllers dialog: Align text in controllers dialog
- lang: a few of lang strings improve

### Before
image

### After
image

please Mr.@Zangetsu38 and Mr.@bookmist check this PR

### CODE REVIEW

File: lang/ru.xml [hunk 1]
```diff
@@ -240,7 +240,7 @@
 	</content_manager>
 
 	<controllers name="Контроллеры">
-		<connected>Контроллеры подключены</connected>
+		<connected>{} контроллеры подключены</connected>
```
Comment (bookmist) [inline lang/ru.xml:243]: <connected>Контроллеров подключено: {}</connected>
@KaneDbD what do you think?

Comment (KaneDbD) [reply lang/ru.xml:243]: yeah, would be better ig

Comment (Croden1999) [reply lang/ru.xml]: done

**Review by bookmist — DISMISSED** [review]

File: lang/ru.xml [hunk 2]
```diff
@@ -597,7 +598,6 @@
 	<welcome name="Добро пожаловать в Vita3K">
 		<line_first>Vita3K, эмулятор PlayStation Vita</line_first>
 		<line_second>Vita3K - это эмулятор PlayStation Vita с открытым исходным кодом, написанный на C++ для Windows, Linux, macOS и Android.</line_second>
-		<line_third>Эмулятор до сих пор находится в ранних стадиях, мы очень ценим любые отзывы и тесты.</line_third>
```
Comment (Croden1999) [inline lang/ru.xml:600]: please Mr. @bookmist check this line

File: vita3k/gui/src/user_management.cpp [hunk 3]
```diff
@@ -326,7 +326,7 @@ void draw_user_management(GuiState &gui, EmuEnvState &emuenv) {
             }
             user_id_selected = fmt::format("{:0>2d}", id);
             auto i = 1;
-            const auto user = lang["user"].c_str();
+            const auto user = "User";
```
Comment (Zangetsu38) [inline vita3k/gui/src/user_management.cpp:329]: revert

Comment (Croden1999) [reply vita3k/gui/src/user_management.cpp:329]: > revert

actually i revert my change, same like ps4

Comment (Zangetsu38) [reply vita3k/gui/src/user_management.cpp:329]: ? In french is writer no user, but "Utilisateur1"

File: lang/ru.xml (hunk 2 above)
Comment (bookmist) [reply lang/ru.xml:600]: `Эмулятор до сих пор находится в разработке, мы очень ценим любые отзывы и тесты.`
@KaneDbD What do you think?

Comment (KaneDbD) [reply lang/ru.xml:600]: 👍

File: vita3k/gui/src/user_management.cpp (hunk 3 above)
Comment (Croden1999) [reply vita3k/gui/src/user_management.cpp:329]: > ? In french is writer no user, but "Utilisateur1"

ahhh, yes, u r right Mr. Zangetsu, i check again and exist on ps4, only jpn/kor/chn is "User1"

File: lang/ru.xml (hunk 2 above)
Comment (Croden1999) [reply lang/ru.xml:600]: done

File: lang/tr.xml [hunk 4]
```diff
@@ -367,7 +364,6 @@ Devam etmek istediğinizden emin misiniz?</user_delete_warn>
 		<change_avatar>Avatar Değiştir</change_avatar>
 		<reset_avatar>Avatar Sıfırla</reset_avatar>
 		<name>Ad</name>
-		<user>Kullanıcı</user>
```
Comment (Zangetsu38) [inline lang/tr.xml:370]: Revert

Comment (Croden1999) [reply lang/tr.xml:370]: here is display "User1"

Comment (Zangetsu38) [reply lang/tr.xml:370]: Ah 🤔 ok

**Review by Zangetsu38 — APPROVED** [review]

File: lang/tr.xml (hunk 4 above)
Comment (Zangetsu38) [reply lang/tr.xml:370]: Is ps4 that ? Is on what device ?

Comment (Croden1999) [reply lang/tr.xml:370]: yes, is on ps4, i used remote play on my vita

Comment (Zangetsu38) [reply lang/tr.xml:370]: ah is vita lol

Comment (Zangetsu38) [reply lang/tr.xml:370]: you can show me same screen in french ? i am curius for one things here

Comment (Croden1999) [reply lang/tr.xml:370]: > ah is vita lol

yes, Mr. Zangetsu XD

Comment (Croden1999) [reply lang/tr.xml:370]: > you can show me same screen in french ? i am curius for one things here

sure, give me a second

**Pull request merged**